devlib/target.py

```diff
@@ -1041,6 +1041,12 @@ def adb_reboot_bootloader(self):
     def adb_root(self):
         adb_command(self.adb_name, 'root', timeout=30)
 
+    def usb_charge(self, enable=True, sysfs_path=None):
```

Comment:
Nor really sure how much standard is that path... but AFAIK is the only one we found so far.
Should we keep it hardcoded in the method, for the time being, and refactor it better once we should find other possible options?